src/box/lua/config/applier/credentials.lua

```diff
@@ -1,30 +1,326 @@
+            add_permission('universe', '', 'usage', res)
+
+        elseif name == 'admin' then
+            add_permission('universe', '', 'read', res)
+            add_permission('universe', '', 'write', res)
+            add_permission('universe', '', 'execute', res)
+            -- TODO add when #8906 is merged:
+            -- add_permission('universe', '', 'lua_eval', res)
+            -- add_permission('universe', '', 'lua_call', res)
+            -- add_permission('universe', '', 'sql', res)
+            add_permission('universe', '', 'session', res)
+            add_permission('universe', '', 'usage', res)
+            add_permission('universe', '', 'create', res)
+            add_permission('universe', '', 'drop', res)
+            add_permission('universe', '', 'alter', res)
+            add_permission('universe', '', 'reference', res)
+            add_permission('universe', '', 'trigger', res)
+            add_permission('universe', '', 'insert', res)
+            add_permission('universe', '', 'update', res)
+            add_permission('universe', '', 'delete', res)
+
+        else
+            -- Newly created user:
+            add_permission('role', 'public', 'execute', res)
+
+            add_permission('universe', '', 'session', res)
+            add_permission('universe', '', 'usage', res)
+
+            add_permission('user', name, 'alter', res)
+        end
+
+    elseif role_or_user == 'role' then
+        -- luacheck: ignore 542 empty if branch
+        if name == 'public' then
+            add_permission('function', 'box.schema.user.info', 'execute', res)
+            add_permission('function', 'LUA', 'read', res)
+
+            add_permission('space', '_vcollation', 'read', res)
+            add_permission('space', '_vspace', 'read', res)
+            add_permission('space', '_vsequence', 'read', res)
+            add_permission('space', '_vindex', 'read', res)
+            add_permission('space', '_vfunc', 'read', res)
+            add_permission('space', '_vuser', 'read', res)
+            add_permission('space', '_vpriv', 'read', res)
+            add_permission('space', '_vspace_sequence', 'read', res)
+
+            add_permission('space', '_truncate', 'write', res)
+
+            add_permission('space', '_session_settings', 'read', res)
+            add_permission('space', '_session_settings', 'write', res)
+
+        elseif name == 'replication' then
+            add_permission('space', '_cluster', 'write', res)
+            add_permission('universe', '', 'read', res)
+
+        elseif name == 'super' then
+            add_permission('universe', '', 'read', res)
+            add_permission('universe', '', 'write', res)
+            add_permission('universe', '', 'execute', res)
+            -- TODO add when #8906 is merged:
+            -- add_permission('universe', '', 'lua_eval', res)
+            -- add_permission('universe', '', 'lua_call', res)
+            -- add_permission('universe', '', 'sql', res)
```

Comment:
Done, same for `'admin'` case

Reply:
Ok, it was caught by my `add_defaults()` test, neither `'super'` nor `'admin'` don't have this new permissions in box, because they are already covered by `'execute'`.

Reply:
I suggest we remove them here too.

Reply:
OK.